packages/nx-monorepo/src/projects/typescript/nx-monorepo.ts

```diff
@@ -133,7 +133,7 @@ interface OverrideNxBuildTaskOptions {
  * This project type will bootstrap a NX based monorepo with support for polygot
  * builds, build caching, dependency graph visualization and much more.
  *
- * @pjid nx-monorepo
+ * @pjid nx-monorepo-ts
```

Comment:
Might need to consider this a breaking change? If you can have multiple `pjid`s maybe we can also keep `nx-monorepo`?

Reply:
its only used on initial bootstrap so not a breaking change technically.